.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules/
 # Logs
 logs
 *.log
+!fake-file.log
```

Comment:
📝 This file was being excluded by the `gitignore` above it.  We need to explicitly unignore it.